modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -181,12 +180,14 @@ export default {
       autofocus: this.autofocus,
       onUpdate: this.editorUpdate,
       extensions: [
-        StarterKit,
+        StarterKit.configure({
+          document: false,
+          heading: false
+        }),
```

Comment:
remove duplicated extensions from `StartKit` as they're added on our side via `this.aposTiptapExtensions`.

fixes

![image](https://user-images.githubusercontent.com/8301962/192538989-bacce208-f3dc-4566-8e10-2e0c250d9248.png)